Sources/Build/BuildPlan/BuildPlan+Product.swift

```diff
@@ -50,19 +56,22 @@ extension BuildPlan {
             }
         }
 
-        // Link C++ if needed.
-        // Note: This will come from build settings in future.
-        for target in dependencies.staticTargets {
-            if case let target as ClangTarget = target.underlying, target.isCXX {
-                let triple = buildProduct.buildParameters.triple
-                if triple.isDarwin() {
-                    buildProduct.additionalFlags += ["-lc++"]
-                } else if triple.isWindows() {
-                    // Don't link any C++ library.
-                } else {
-                    buildProduct.additionalFlags += ["-lstdc++"]
+        // Don't link unavailable libc++ or libstd++ when building for Embedded Swift
+        if !buildProduct.product.targets.contains(where: \.underlying.isEmbeddedSwiftTarget) {
```

Comment:
I don't think C++ runtime library is unavailable for all Embedded platforms. For example, it seems like Pico SDK provides libc++: https://github.com/ARM-software/LLVM-embedded-toolchain-for-Arm

So it might be worth mentioning that in such case we ask users to explicitly specify desired C++ runtime via `Xlinker`.

Reply:
I've expanded the comment removing the word "unavailable", with potential solutions suggested there as well.